01/app.js

```diff
@@ -0,0 +1,11 @@
+const getButton = document.querySelector('button')
+
+// getButton.addEventListener('click', e => {
+//     console.log('clicked')
+// })
+
+const text = function() {
+    console.log('clicked')
+};
+
+getButton.addEventListener('click', text);
```

Comment:
Warto sprawdzić czy `getButto`n nie jest `null` - zabezpieczenie przed błędem

ps. Nazwa `getButton` byłaby odpowiednia dla funkcji bo wskazuje na działanie tj. zawiera czasownik. 
Ta zmienna to element więc powinien być to rzeczownik tj. `button` czy `buttonElement`